include/grpc++/server_builder.h

```diff
@@ -170,6 +178,28 @@ class ServerBuilder {
     int* selected_port;
   };
 
+  struct SyncServerSettings {
+    SyncServerSettings()
+        : num_cqs(GPR_MAX(gpr_cpu_num_cores(), 4)),
+          min_pollers(1),
+          max_pollers(INT_MAX),
+          cq_timeout_msec(1000) {}
+
```

Comment:
Just curious why you don't use a `SyncServerSettings` as the argument to `Server::Server` so that it is ready for future expansion (if needed)?